Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -449,6 +468,28 @@ public function fulltext($searchWord) {
 	}
 
 	/**
+	 * Wrap search term matches found in the result for highlighting purposes. Only available for fulltext searches.
```

Comment:
can't we just simply *ALWAYS* run highlight requests as soon as a fulltext query is involved? Would keep the API in sync with SimpleSearch.